backend/ohq/views.py

```diff
@@ -237,6 +237,45 @@ def list(self, request, *args, **kwargs):
         membership.save()
         return super().list(request, *args, **kwargs)
 
+    def create(self, request, *args, **kwargs):
+        """
+        Create a new question and check if it follows the rate limit
+        """
+
+        queue = Queue.objects.get(id=self.kwargs["queue_pk"])
+        if (
+            queue.rate_limit_length is not None
```

Comment:
I don't know how the frontend will eventually try to change the rate limit, but we my have to do a different check than not None (Maybe a `!= -1` or something. ee can figure that out as the frontend gets built for this

Reply:
Since the rate limit field triple comes together, would be the easiest if we have another `rate_limited_enabled` boolean field.